src/Summoner/Config.hs

```diff
@@ -136,6 +139,9 @@ configT = Config
     license :: Key -> BiToml License
     license =  dimapBijection unLicense License . Toml.str
 
+    extensions :: Key -> BiToml [Text]
+    extensions = dimapBijection Just (fromMaybe []) . Toml.maybeP (Toml.arrayOf Toml.strV)
```

Comment:
`maybeToMonoid` is one character lower than `(fromMaybe [])` :slightly_smiling_face: